Simplenote/src/main/java/com/automattic/simplenote/utils/HtmlCompat.java

```diff
@@ -0,0 +1,16 @@
+package com.automattic.simplenote.utils;
+
+import android.os.Build;
+import android.text.Html;
+import android.text.Spanned;
+
+public class HtmlCompat {
+    @SuppressWarnings("deprecation")
+    public static Spanned fromHtml(String source) {
```

Comment:
This change isn't really required. I just added it to avoid using now deprecated method and eliminate all warnings. 

In N they added this new fromHtml method which takes flags parameter that allows for changing how text rendering works. 

I linked few references in description of this commit https://github.com/Tunous/simplenote-android/commit/c7ae2fac26d3a35a54d8ca38c271c6a39378eea9. 

I can revert if that's not wanted.